contracts/interfaces/IRelayHub.sol

```diff
@@ -41,12 +41,12 @@ interface IRelayHub {
     // The actual relayed call was not executed, and the recipient not charged.
     // The reason field contains an error code: values 1-10 correspond to CanRelayStatus entries, and values over 10
     // are custom recipient error codes returned from acceptRelayedCall.
-    event CanRelayFailed(
+    event UnpaidPaymasterRejection(
```

Comment:
This event is matched again the TransactionRelayed. they should matching indexed params.

Reply:
For idealogical reasons? Failure and success may need to highlight different aspects of a transaction. In this case, it may be critical to be able to easily get all transactions rejected by a specific paymaster.